Fhl a11y fixes

sugupta427 opened the pull request:
Fixing A11y issue as FHL project

mttwc commented:
Oops, forgot to add UTs. Will add them in a sec.

iconix commented on src/scripts/clipperUI/panels/optionsPanel.tsx:
```diff
@@ -30,7 +30,7 @@ class OptionsPanelClass extends ComponentBase<{ }, OptionsPanelProps> {
 				<div id={Constants.Ids.clipButtonContainer} className="wideButtonContainer">
 					{clipButtonEnabled
-					?	<a id={Constants.Ids.clipButton} className="wideActionButton" {...this.enableInvoke(this.props.onStartClip, 0) }>
+					?	<a id={Constants.Ids.clipButton} className="wideActionButton" {...this.enableInvoke(this.props.onStartClip, 1) }>
```
I assumed when Dipak made this tab index 0, it was because there was already a 1 and he didn't feel like bumping all the other tab indices down :P Not true?

mttwc replied:
Nope, this is not the case, he made the change to 0 as he thought it was the lowest viable number. Turns out it's 1!

With our tabbing, to avoid huge refactors when changing tab orders, we start each grouping at a multiple of 10, rather than having one directly after the previous (e.g., 1, 2, 3, ... 55, 56, 57 etc)

iconix replied:
kewl :)

iconix commented:
:sheep:

leakymemory commented:
:+1:

vadobre committed:
Day1 fixes
+ Fixed Edge loading
+ Fixed Intial focus and hide some Images that are not needed

vadobre committed:
+ Fix initial focus for main view, by focusing selected list option

vadobre committed:
+ Added top level a11y grouping for each visible pane

vadobre committed:
Fixed FullPage Link UIA Tree issues

vadobre committed:
Fixed Location string UIA issues

vadobre committed:
Fix UIA issues for footer

vadobre committed:
Fixed Loading indicator to take focus, show focus and have correct accessible info

sugupta427 committed:
hiding font-up and font-down images which are not needed

sugupta427 committed:
updating the string for pageview

sugupta427 committed:
Merge branch 'master' into FHL-A11y-Fixes
